src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>to do list</title>
+    <script
+      src="https://kit.fontawesome.com/69c7e163a7.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div id="container">
+      <ul id="menu">
+        <li id="title"><p>Today's to do</p></li>
+        <li>
+          <input
+            type="text"
+            name=""
+            id="task"
+            onfocus="this.value=''"
```

Comment:
Your input holds the previous value while you do not click on it. You could add it to your eventListener on DOM. When the new task was added it resets itself to an empty string.